addons/common/functions/fnc_player.sqf

```diff
@@ -4,7 +4,7 @@
  * Use this in INIT and RESPAWN eh scripts, because ACE_player isn't reset yet.
  *
  * Arguments:
- * NONE.
+ * None.
```

Comment:
Thats not mine dot. I just mass replace `NONE` to `None`. The dot was their before i touched it. Blame @commy2 for it :P